aries_cloudagent/vc/ld_proofs/constants.py

```diff
@@ -14,3 +14,7 @@
 
 VERIFIABLE_CREDENTIAL_TYPE = "VerifiableCredential"
 VERIFIABLE_PRESENTATION_TYPE = "VerifiableCredential"
```

Comment:
One last note. This should be `VerifiablePresentation` (this is my fault, so I can also fix it in a separate PR. It doesn't seem to be used atm)


```suggestion
VERIFIABLE_PRESENTATION_TYPE = "VerifiablePresentation"
```

Reply:
I am going to follow this PR merge with a different PR which includes a different mechanism for prover/holder to provide list of `record_ids` for each `input_descriptor`, I will make this change in that PR, if it is ok.